docs/config.md

```diff
@@ -599,6 +600,10 @@ Type: `bool`
 **REMOVED:**  this is not configurable any more
 in the [new mDNS implementation](https://github.com/libp2p/zeroconf#readme).
 
+## `Experimental`
+
+Toggle and configure experimental features of Kubo. Experimental features are listed [here](https://github.com/ipfs/kubo/blob/master/docs/experimental-features.md).
```

Comment:
```suggestion
Toggle and configure experimental features of Kubo. Experimental features are listed [here](./experimental-features.md).
```

Should be able to load locally, don't think we need an absolute link here.

Text seems good enough, despite some experimental features being in this document as well 🤦